Update/_tsum_001.txt

```diff
@@ -2701,7 +2701,7 @@ void main()
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fcdb77>沙都子</color>", NULL, "<color=#fcdb77>Satoko</color>", NULL, Line_ContinueAfterTyping); }
 	ModPlayVoiceLS(4, 4, "ps3/s09/04/500400651", 128, TRUE);
 	OutputLine(NULL, "「私、いつも言ってましてよ？",
-		   NULL, "\"Didn't I always tell you so?", Line_WaitForInput);
```

Comment:
@Norgus 
Bothering you with more translation questions. What do you think of this sentence?

Reply:
I think the common expression might be "it's like I always say", so I kind of feel like the original translation is closer, if a little awkward sounding.

Reply:
I see. By original, do you mean "I told you." ?